apis/beacon/blobs/blobs.yaml

```diff
@@ -0,0 +1,62 @@
+get:
+  operationId: getBlobs
+  summary: Get blobs
+  description: |
+    Retrieves blobs for a given block id.
+    Depending on `Accept` header it can be returned either as json or as bytes serialized by SSZ.
+    It is recommended that blob retrieval for finalized blocks should by done by slot else the underlying index might not be available.
+  tags:
+    - Beacon
+  parameters:
+    - name: block_id
+      in: path
+      required: true
+      $ref: '../../../beacon-node-oapi.yaml#/components/parameters/BlockId'
+    - name: indices
+      in: query
+      description: Array of indices for blobs to request for in the specified block. Returns all blobs in the block if not speicfied.
+      required: false
+      schema:
+        type: array
+        uniqueItems: true
+        items:
+          $ref: '../../../beacon-node-oapi.yaml#/components/schemas/Uint64'
+  responses:
+    "200":
+      description: "Successful response"
+      content:
+        application/json:
+          schema:
+            title: GetBlobsResponse
+            type: object
+            properties:
+              data:
+                type: array
+                items:
+                  oneOf:
+                    - $ref: "../../../beacon-node-oapi.yaml#/components/schemas/Deneb.BlobSidecar"
+                minItems: 0
+                maxItems: 4
+        application/octet-stream:
+          schema:
+            description: "SSZ serialized block bytes. Use Accept header to choose this response type"
```

Comment:
what's this returning?
My assumption is that its an ssz array of blobs in the order specified in the query?

eg. if i set `?indicies=1,0` do i get an ssz array of blobs, with blob 1 first, and blob 0 second?

Reply:
Thanks for raising this, great point - I think it would be good to be consistent with other endpoints that accepts array type query params (e.g. `getStateValidators), i.e. not guaranteeing order, unless there are specific reasons for guaranteeing the order when indices are specified?

From `getStateValidators`:
> If an index or public key does not
    match any known validator, no information will be returned but this will not cause an error.  There are no guarantees for the
    returned data in terms of ordering

Reply:
i haven't looked at the SSZ, hopefully its easy to get the blob id from the object.. assuming thats fine...

Reply:
Added more details in description to specify the behaviour when `indices` is specified, and fixed the SSZ response description. Commit: 354ff9fa9b80ff2fd68bc5bfdabe2fe3f85b25f1.

@rolfyone what do you mean by getting the blob id from the object?

Reply:
just wondering if the ordering is important, so if i query some weird order, do i need to process them in some specific order, and is that possible based on the return data if we're not returning in the same order...
I'm not sure just asking the stupid question to see if its going to cause problems when ppl get blobs back in a different order to the id's they passed...

Reply:
I'm not sure if the ordering is important tbh, but we do have `index` in each blob sidecar, which seems to be a more reliable property for user to depend on, and consumers should probably be encouraged to use this for ordering instead.